ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/filter/role/RoleAuditFilter.java

```diff
@@ -94,15 +95,8 @@ private boolean isFilteredUnchecked(AuditEntry logEntry)
     {
         Set<RoleResource> roles = getRoles(logEntry.getUser());
         List<IResource> operationResourceHierarchy = getResourceHierarchy(logEntry.getResource());
-        for (Permission operation : logEntry.getPermissions())
-        {
-            if(!isOperationWhitelistedOnResourceByRoles(operation, operationResourceHierarchy, roles))
-            {
-                return false;
-            }
-        }
-
-        return true;
+        return logEntry.getPermissions().stream()
```

Comment:
When looking at this patch set I realize that the line above (call to getResourceHierarchy()) can be replaced with Resources.chain() (part of Cassandra code base).

That way we'd get rid of this custom function all together.